lattice/header_entries.py

```diff
@@ -611,6 +614,36 @@ def translate(self, input_file_path, container_class_name, schema_base_class_nam
                 # objects might be included and used from elsewhere.
                 ObjectSerializationDeclaration(base_level_tag, self._namespace)
 
+    # .............................................................................................
```

Comment:
This really should use the new Schema class!

Reply:
Not yet. Next PR.